board/safety/safety_volkswagen.h

```diff
@@ -1,142 +1,232 @@
-// Safety-relevant CAN messages for the Volkswagen MQB platform.
-#define MSG_EPS_01              0x09F
-#define MSG_MOTOR_20            0x121
-#define MSG_ACC_06              0x122
-#define MSG_HCA_01              0x126
-#define MSG_GRA_ACC_01          0x12B
-#define MSG_LDW_02              0x397
-
-const int VOLKSWAGEN_MAX_STEER = 250;               // 2.5 Nm (EPS side max of 3.0Nm with fault if violated)
+// Safety-relevant steering constants for Volkswagen
+const int VOLKSWAGEN_MAX_STEER = 300;               // 3.0 Nm (EPS side max of 3.0Nm with fault if violated)
```

Comment:
I'm using this opportunity to ask for reconsideration of the 2.5Nm cap. @rbiasini and I talked about this over the phone about six weeks ago, and the outcome was an agreement to talk more about it at a later date. It's later, so here I am.

I've tried it both ways, and the difference between 2.5Nm and 3.0Nm makes a definite, meaningful difference in regular highway driving disengagements. 3.0Nm is what the factory LKAS camera sends, so VW didn't feel it was unsafe. I don't feel that 3.0Nm was a problem to override. With the ramp-rate limit, the extra torque takes 1.25-1.50 secs to come in, so it isn't a lane-departure concern under Comma's safety model.
 
My goal is to have all BP/C2 Volkswagen drivers using the signed Comma safety. Actually I don't want to have a "community" fork at all, but one step at a time. Having made the ACC_06->TSK_06 change in this PR, the torque limit is the last thing standing in the way for the safety part of the job.

Reply:
Drove quite a bit with 2.5Nm lately and I agree 3Nm seems more appropriate and still on par with Toyota level torque.
Can we make a PR with just this change? It also involves a change to openpilot.

Reply:
Can do.